test/Interop/Cxx/class/access/swiftify-private-fileid-executable.swift

```diff
@@ -0,0 +1,19 @@
+// RUN: rm -rf %t
+// RUN: split-file %s %t
+
+// RUN: %target-swift-frontend -emit-ir -plugin-path %swift-plugin-dir %t/blessed.swift -module-name main -I %S/Inputs -o %t/out -Xcc -std=c++20  -cxx-interoperability-mode=default -enable-experimental-feature SafeInteropWrappers -verify
+
+// REQUIRES: swift_feature_SafeInteropWrappers
+
+// FIXME swift-ci linux tests do not support std::span
+// UNSUPPORTED: OS=linux-gnu, OS=linux-android, OS=linux-androideabi
+
+//--- blessed.swift
```

Comment:
suggestion: could inline header and module file if we're using `split-file` anyways. Or simply `cp %s %t/blessed.swift`. Feel free to ignore.